action.yml

```diff
@@ -26,4 +26,4 @@ inputs:
     description: Whether new commits to the PR should re-deploy the Fly app
     default: true
   secrets:
-    description: Secrets to be set on the app. Separate multiple secrets with a space, i.e., `FIRST_SECRET=${{ secrets.FIRST_SECRET }} SECOND_SECRET=${{ secrets.SECOND_SECRET }}`
+    description: Secrets to be set on the app. Separate multiple secrets with a space
```

Comment:
`${{ secrets.FIRST_SECRET }}` broke the action

Reply:
How should I set the secret now?

Reply:
How should I set the secrets? I test it with `secrets: DATASET=staging TEST_SECRET=helloworld` but it doesn't seem to work

Reply:
Exactly how it is described. It Works for me.

I think we should change it to  passing it as a proper key value where each secret is passed separately.

Reply:
We could update the README inputs section to include the `secrets` and examples of how to use it.  Strangely, it doesn't work for me. Maybe I should create an issue.

```
  - name: 🚀 Deploy Preview
        id: deploy
        uses: superfly/fly-pr-review-apps@1.0.0
        with:
          path: "./web"
          name: deploy-preview-${{ github.event.number }}--${{ steps.app_name.outputs.value }}
          region: fra
          image: "registry.fly.io/${{ steps.app_name.outputs.value }}:${{ needs.extract_branch_name.outputs.value }}-${{ github.sha }}"
          secrets: SANITY_DATASET=staging
```

Reply:
@HuyAms to check against my branch do `uses: faros-ai/fly-pr-review-apps@mt/app-name`

Reply:
`faros-ai/fly-pr-review-apps@mt/app-name` works for me. You guys made my day 🙌 @tovbinm @alexslade